java/recipes/default.rb

```diff
@@ -1,2 +1,6 @@
-package 'openjdk-6-jre-headless'
-package 'openjdk-6-jre-lib'
+if node['lsb']['codename'] == 'xenial'
```

Comment:
This is unrelated when I thought Java was needed and xenial needs these packages.

Figured I would push it because we will need it later.